Lib/unittest/case.py

```diff
@@ -390,6 +404,8 @@ class TestCase(object):
 
     _classSetupFailed = False
 
+    _class_cleanups = []
```

Comment:
Minor nit (maybe!): In this case, the blank line between these two class attributes may not be needed.